### PR TITLE
fix(icon): Fix icon color references

### PR DESCRIPTION
**Pull request by anicholls**
## Summary

I missed a few things in #706. This addresses those issues.

### CODE REVIEW

File: modules/icon/css/README.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ Default: `24`
 ## Default Options
 
 ```js
-initializeIcons((iconPath = null), (selector = '.wdc-icon'), iconRoot = document);
+initializeIcons((iconPath = null), (selector = '.wdc-icon'), (iconRoot = document));
```
Comment (lychyi) [inline modules/icon/css/README.md:256]: 😬

Comment (anicholls) [reply modules/icon/css/README.md:256]: The linter did this, not me!

**Comment by lychyi** [comment]
@anicholls If this addresses #706, I'm not clear on why this PR is against `master`? Are some things broken today that we need to address before we merge `master` into `prerelease/v4`?

The resulting colors seem to be the same regardless of whether we use `primary.` or `iconColors.` right?

**Comment by anicholls** [comment]
@lychyi Oh wow. Major brain fart. Sorry, I will change the base to prerelease.

> The resulting colors seem to be the same regardless of whether we use `primary.` or `iconColors.` right?
Yes, however `primary` no longer exists in 2.0

**Pull request merged**